### ISSUE TITLE
Failed to parse IMG file from SMART-1  mission.

### ISSUE THREAD

**Issue by BlockoS** [BlockoS/tinypds#2]
Parsing AMI_ER3_041113_00024_00600.IMG failed with :
> line 77:61: invalid element separator
> DARK_CURRENT_FILE_NAME         = (AMI_CMA_071101_00001_00000.IMG,             

Data can be found here : ftp://psa.esac.esa.int/pub/mirror/SMALL-MISSIONS-FOR-ADVANCED-RESEARCH-AND-TECHNOLOGY/AMIE/S1-L-X-AMIE-3-RDR-EEP-V1.0/DATA/EARTH_ESCAPE_2004_10_TO_11/EARTH_ESCAPE_041113/

**Comment by BlockoS** [comment]
According to the spec: `sequence_1d : : = (scalar_value [, scalar_value]*)` with 
- `scalar_value := numeric_value | date_time_value | text_string_value | symbolic_value`.
- `text_string_value := quoted_text`
- `symbolic_value := identifier | quoted_symbol`
- `quoted_text := “[character]*”`
- `identifier := letter [letter | digit | _letter | _digit ]*` 
- `quoted_symbol := ‘[character]+’`
- `letter := A-Z | a-z`
- `digit := 0|1|2|3|4|5|6|7|8|9`

**tl;dr** The file is not a valid PDS3 file. The set values should have been quoted... 🤦‍

**Comment by BlockoS** [comment]
They are 3.7 files... The implementation is based upon version 3.8.
